docs/src/languages/php/_index.md

```diff
@@ -304,16 +210,197 @@ markdownify=false
 
 {{< /codetabs >}}
 
-
-
 {{% config-reader %}}
 [`platformsh/config-reader` Composer library](https://github.com/platformsh/config-reader-php)
 {{% /config-reader%}}
 
-## Runtime configuration
+## PHP settings
+
+You can configure your PHP-FPM runtime configuration by specifying the [runtime in your app configuration](../../create-apps/app-reference.md#runtime).
+
+You can set the [PHP runtime timezone](../../create-apps/timezone.md).
```

Comment:
:lady_beetle: I just realized that this doesn't really fit here. The first sentence relates to the table below, but this breaks it up as is unrelated. It's not defined in the same place and so should be separated.